### ISSUE TITLE
Aria-label missing for users in user list (or user ID for eg CSS override etc)

### ISSUE THREAD

**Issue by MWWorks** [bigbluebutton/bigbluebutton#24197]
In 2.7, the elements in the user list had an **aria-label** attribute with the user's name.

I had been using this to target elements to apply CSS overrides to users (changing colours for specific roles etc).

This seems to be missing in 3.0(.16), with no apparent way of identifying/targeting any given user in the list.

The previous method of the aria-label worked well as it fulfilled an accessibility role. But ideally there would also be eg a **data-id** attribute containing the (external) user ID.

**Comment by MWWorks** [comment]
Confirm this works as expected - and having the ID in there also is a much cleaner solution. Thanks as ever for the amazing work on BBB!

**Comment by ffdixon** [comment]
Thanks for the kind words.  If you want to help the project, give us a shout out on the social media of your choice.  We want others to know about the good work going on in the BigBlueButton project.
